deploy/flux-deployment.yaml

```diff
@@ -25,6 +25,11 @@ spec:
       - name: git-keygen
         emptyDir:
           medium: Memory
+       
+      # This is holding the reference to docker-credentials if provided from the outside 
+      - name: docker-creds
+        configMap:
```

Comment:
I think this is better as a secret, as for the chart.

Reply:
indeed.